backend/local.graphql.server.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, **kwargs):
 def request_context(headers, mock=False):
```

Comment:
the name of this file doesn't conform to the naming convention used on all other files. is there a reason for that?

Reply:
yeah, all other python files in the project use _ naming. is there a reason this one is local.graphql.server.py and not local_graphql_server.py?